files/default/rules/opensearch.rules.yml

```diff
@@ -0,0 +1,29 @@
+groups:
+- name: opensearch 
+  rules:
+  - alert: YellowCluster 
```

Comment:
my only issue with this metrics is that it will always be yellow on default cluster on managed

Reply:
One more reason to deliver Hopsworks HA. These are files not templates, so we cannot parametrize them. I think it's fine to merge it like this and then when Hopsworks HA hits cloud everything should be fine.

Reply:
ok, just we need to keep it in mind